tasks/static_analysis_suite_check.py

```diff
@@ -45,6 +49,9 @@ async def run_async(
                 == StaticAnalysisSingleFileSnapshotState.CREATED.db_id,
             )
         )
+        metrics_context = MetricContext(
+            repo_id=suite.commit.repository.repoid, commit_id=suite.commit.id
```

Comment:
i am probably overoptimizing, but if you have a repository db model, you can pass in `repo.owner.ownerid` so `MetricContext` doesn't have to query it later

maybe i should write a factory function that takes db models and prepopulates with them